CSharpSourceCode/CampaignSupport/Assimilation/AssimilationCampaignBehavior.cs

```diff
@@ -13,18 +11,26 @@ public class AssimilationCampaignBehavior : CampaignBehaviorBase
         public override void RegisterEvents()
         {
             CampaignEvents.OnGameLoadFinishedEvent.AddNonSerializedListener(this, OnGameLoadFinishedEvent);
```

Comment:
I would suggest to use the OnSessionLaunched event instead. I think GameLoadFinished only fires when loading a savegame, not when starting a new game, and my guess is that you want both.

Reply:
I don't add this component to all settlements at start, but to each settlement after changing its owner and only if culture is different. After loading a save game I need to add already created components to their settlements. That's why I use OnGameLoadingFinishedEvent.

Reply:
OK, good enough :)